### DISCUSSION TITLE
Automatic update of stand-alone notes referenced in workspace note

### DISCUSSION THREAD

**Discussion by pohall63** [windingwind/zotero-better-notes#719] [Q&A] [answered]
Hi,

I am new at this and I might be missing something here. I have tried this on Zotero 6 and 7 but it's not working and I am not sure if it's a feature or not. Here is the scenario:

1. I create a collection with subcollections. In the main collection, I created a workspace note and in subcollections, I created a stand-alone note that is linked to the workspace note. 
2. I import the content from the stand-alone note to the workspace note and this works fine.

I was under the impression that if I update the stand-alone note it would automatically update the workspace note. At the moment I have to reimport the content.

Any suggestions?

Thanks!

**Comment by windingwind** [comment, marked as answer]
The `imported` note content will not be updated automatically- that's why we recommend you use the note link, instead of importing.

Clicking note link always brings you to the latest version of the sub-note.

**Reply by pohall63** [reply]
Thanks, now I know. I wanted to have an updated consolidated overview of the content of my sub-notes. Maybe something to consider?

**Reply by windingwind** [reply]
In workspace, you can click the note link and the sub-note will be opened on the right side.
